kratos/python/add_processes_to_python.cpp

```diff
@@ -238,8 +238,12 @@ void  AddProcessesToPython(pybind11::module& m)
     ;
 
     py::class_<FindConditionsNeighboursProcess, FindConditionsNeighboursProcess::Pointer, Process>(m,"FindConditionsNeighboursProcess")
-            .def(py::init<ModelPart&, int, unsigned int>())
-    .def("ClearNeighbours",&FindConditionsNeighboursProcess::ClearNeighbours)
+        .def(py::init<Model&, Parameters>())
+        .def(py::init<ModelPart&>())
+        .def(py::init<ModelPart&, const int>())
+        .def(py::init<ModelPart&, const int, const unsigned int>())
+        .def("ClearNeighbours",&FindConditionsNeighboursProcess::ClearNeighbours)
```

Comment:
Instead of exposing this one, I'd expose the `Clear`, which is in the base `Process` API. About its implementation, I'd make it call the `ClearNeighbours` so
```c++
void Clear() {
    ClearNeighbours();
}
```
By doing this, we ease eventually removing the `ClearNeighbours`, which doesn't conform with the base class API.

Reply:
Makes sense